components/aws/sagemaker/common/_utils.py

```diff
@@ -82,6 +82,32 @@ def get_component_version():
 
     return component_version
 
+def print_logs_for_job(region, job_name, log_grp):
+    """Gets the cloudwatch logs for sagemaker jobs"""
+    try:
+        logging.info('\n******************** Cloudwatch logs for ' + log_grp + ' ' + job_name + ' ********************\n')
```

Comment:
nit: Standardise formatting using a format string `CLOUDWATCH_SECTION_FMT = '\n****** {} ******\n'` and `logging.info(CLOUDWATCH_SECTION_FMT.format('Cloudwatch logs for ...'))`

Reply:
Also why do we need manual `\n`? Is it just for clearer log formats to put spaces between?

Reply:
yes